code/upaya/app/views/shared/_user_search.html.slim

```diff
@@ -0,0 +1,13 @@
+.panel.panel-default
+
+  = form_tag support_find_path, class: 'form-horizontal', method: 'post', role: 'form' do
+    .panel-body
+      fieldset.form-group
+        legend.solo-legend= t('upaya.headings.search')
+        hr
+        = label_tag :email, nil, class: 'col-md-2 control-label'
+        .col-md-10
+          = email_field_tag :email, params[:email],  class: 'form-control'
+      .form-group
+        .col-md-offset-10.col-md-2
+          = submit_tag 'Search', id: 'user_search', class: 'btn btn-default upaya-button-secondary-blue pull-right'
```

Comment:
I think this may be used, at least let's keep it if it easier than removing it for this first sprint.